package.json

```diff
@@ -4,26 +4,28 @@
   "description": "Two Factor e-mail provider",
   "author": "Roeland Jago Douma <roeland@famdouma.nl>",
   "license": "AGPL-3.0-or-later",
-  "main": "main.js",
+  "main": "index.js",
   "directories": {
     "lib": "lib",
     "test": "tests"
   },
   "scripts": {
-    "dev": "webpack --config webpack.dev.js",
-    "watch": "webpack --progress --watch --config webpack.dev.js",
-    "build": "webpack --progress --hide-modules --config webpack.prod.js",
+    "build": "./node_modules/webpack-cli/bin/cli.js --config js/webpack.prod.config.js",
+    "dev": "./node_modules/webpack-cli/bin/cli.js --config js/webpack.dev.config.js --watch",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "axios": "^0.18.0",
-    "vue": "^2.5.17"
+    "css-loader": "^2.1.1",
+    "jquery": "^3.4.1",
+    "nextcloud_fetch": "^2.1.0",
```

Comment:
We kind of defaulted to nextcloud-axios for network stuff. Also since @ChristophWurst is not maintaining fetch right now. And this will make it do :boom: on older browsers.

Reply:
it's even deprecated via npm. if you install it it should recommend nextcloud-axios instead